### PR TITLE
Test for running hotplug/unplug and load/unload module parallely

### PR DESCRIPTION
**Pull request by PavithraDP**
This test case will stress kernel by loading/unloading the modules and hotplug/unplugging the CPUs parallely and check the kernel robustness.

1)load.py - The test runs loading/unloading modules for certain duration

2)cpu_hotplug.py - The test hotplug/unplugs CPUs for certian duration.

3)em_stress.py - Both scripts runs in parallel to check robustness

### CODE REVIEW

**Comment by PavithraDP** [comment]
@sathnaga

**Comment by PraveenPenguin** [comment]
can load.py and cpu_hotplug.py will be kept under data dir ?

suggestion like 

em_stress.py be initial file (Test trigger )
"**`em_stress.py.data`**"  dir  can contain cpu_hotplug.py and load.py 

you can inspire from Other test 
like https://github.com/avocado-framework/avocado-misc-tests/pull/114

File: em_stress/em_stress.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                P, rc = value.communicate()[0], value.returncode
+                if rc != 0:
+                    print "The test %s failed with return code %d" % (key, rc)
+                    error_count += 1
+        except Exception as e:
+            print("Unexpected error :" + e.message)
+
+        if error_count != 0:
+            self.error('The test failed with errors')
+
```
Comment (PraveenPenguin) [inline em_stress/em_stress.py:43]: can this self.error replaced to self.fail() as self.fail will prompt as 'TestFail:'

File: em_stress/em_stress.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
```
Comment (PraveenPenguin) [inline em_stress/em_stress.py:2]: please add  LICENSE heading , You can inspire from other test

**Comment by PraveenPenguin** [comment]
can you please provide job log , as in ubuntu NV i am not able to run this test as module is not loadable .

**Comment by apahim** [comment]
This repository is frozen. Please send the PR with the requested changes to:
https://github.com/avocado-framework-tests/avocado-misc-tests

**Pull request closed without merging**